Survey/Survey/Sources/Data/Models/LoginTokenApi.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoginTokenApi.swift
+//  Survey
+//
+//  Created by Taher on 4/11/22.
+//
+
+import Foundation
+
+struct LoginTokenApi: Decodable, LoginToken {
```

Comment:
We're using [JSONAPI](https://jsonapi.org/) as the standard for all our projects. What do you think of using our [JSONMapper](https://github.com/nimblehq/JSONMapper)?

Reply:
@ducbm051291 nice suggestion, I didn't aware of that. I created a new [issue](https://github.com/Shayokh144/ic-survey-swiftui-combine/issues/9) for next sprint as there are some remaining task in current sprint.